atd-vze/src/queries/gqlAbstract.js

```diff
@@ -307,7 +307,7 @@ gqlAbstractTableAggregateName (
         return `${dateValue}`;
       }
       case "currency": {
-        return `$${value.toLocaleString()}`;
+        return `$${parseFloat(value).toLocaleString()}`;
```

Comment:
this wasn't working previously cuz `value` is a string type so first we gotta parse it and return as a float :p

Reply:
ah good idea! just pushed that up. yeah wrangling helpers would be good!